### PR TITLE
Changed boost build to B2

### PR DESCRIPTION
**Pull request by PeterTurcan**
Changed boost.build to B2.
Changed Boost:build[] to Boost.build[B2] - seems to work fine with B2 as the text and link

### CODE REVIEW

File: user-guide/modules/ROOT/pages/getting-started.adoc [hunk 1]
```diff
@@ -488,11 +488,11 @@ cmake --build . --target install --config Debug
 cmake --build . --target install --config Release
 ----
 
-Although both the boost:build[] and the CMake scripts work fine, the final installed libraries will have different layouts for historical reasons.
+Although both the boost:build[B2] and the CMake scripts work fine, the final installed libraries will have different layouts for historical reasons.
 Both the CMake integration scripts and the libraries will be installed under different paths and this can create incompatibilities between Boost installations.
 
 TIP: You don't need to *build* Boost with CMake to be able to *use* Boost with CMake.
-CMake integration will work fine even if you install Boost with boost:build[].
+CMake integration will work fine even if you install Boost with boost:build[B2].
```
Comment (alandefreitas) [inline user-guide/modules/ROOT/pages/getting-started.adoc:495]: Rene also requested the links to point somewhere else.

**Review by alandefreitas — CHANGES_REQUESTED** [review]
Commit messages are usually in the present tense:

[Changed boost build to B2](https://github.com/cppalliance/site-docs/pull/127/commits/73fb5db4b89ff0f0905b86aa3061cfd0afc9b130) -> "Change Boost.Build to B2" or something

File: user-guide/modules/ROOT/pages/getting-started.adoc (hunk 1 above)
Comment (PeterTurcan) [reply user-guide/modules/ROOT/pages/getting-started.adoc]: @alandefreitas @grafikrobot 
From the Issue looks like boost:build[] should be replaced with:    https://www.bfgroup.xyz/b2/[B2]
I have done this for the getting started section - and seems to work - but please verifiy ....

Comment (alandefreitas) [reply user-guide/modules/ROOT/pages/getting-started.adoc]: Yes. That looks correct.

**Comment by PeterTurcan** [comment]
@alandefreitas 
Squashed and gtg

**Comment by alandefreitas** [comment]
> Squashed and gtg

Please change the commit msgs to present tense next time

**Pull request merged**